### DISCUSSION TITLE
no master server reply?

### DISCUSSION THREAD

**Discussion by Blindman2** [redeclipse/base#1417] [Help and Support]
Keep getting 'master server not replying no matter the version, any ideas?

**Comment by dc-redeclipse** [comment]
Hello @Blindman2
are you trying to host a server accessible to the public?
I asking this because its a typical replay when a port is blocking your outgoing request

**Reply by Blindman2** [reply]
No sir, just play, checked router and fw, neither appear to be blocking those ports.

**Reply by dc-redeclipse** [reply]
are you using the Steam version?

**Reply by Blindman2** [reply]
ubuntu repository, but even from scratch, same thing.


"PING play.redeclipse.net (159.203.69.7) 56(84) bytes of data.
64 bytes from icculus.org (159.203.69.7): icmp_seq=1 ttl=54 time=10.4 ms
64 bytes from icculus.org (159.203.69.7): icmp_seq=2 ttl=54 time=9.82 ms
64 bytes from icculus.org (159.203.69.7): icmp_seq=3 ttl=54 time=8.69 ms
64 bytes from icculus.org (159.203.69.7): icmp_seq=4 ttl=54 time=8.68 ms
64 bytes from icculus.org (159.203.69.7): icmp_seq=5 ttl=54 time=10.8 ms"

**Reply by dc-redeclipse** [reply]
ok which version are you playing 2.0?

**Reply by dc-redeclipse** [reply]
and can you ping play.redeclipse.net from your machine?

**Reply by Blindman2** [reply]
i can ping it yes. 1.6 , but 2.0 did the same thing

**Reply by dc-redeclipse** [reply]
is port 28800 and 28801 usable for the game can you ensure its not used by a different application?

**Reply by dc-redeclipse** [reply]
and if you got Steam just try the Steam version [https://www.redeclipse.net/steam](https://www.redeclipse.net/steam)
I am very sure its ubuntus fault. Ubuntu is sometimes terrible to users

**Reply by Blindman2** [reply]
So ive noticed, thanks.

**Reply by dc-redeclipse** [reply]
> So ive noticed, thanks.

@Blindman2 so did you solved it? Does Steam works for you?

**Reply by Blindman2** [reply]
yea, turns out i forgot i had two nic cards going as well

**Reply by dc-redeclipse** [reply]
Great happy fragging!
Btw. if you are looking for other players in Red Eclipse, just join us on Discord. 
[https://www.redeclipse.net/discord](https://www.redeclipse.net/discord)
